### DISCUSSION TITLE
Is there an option to access a list of "Recently opened files"?

### DISCUSSION THREAD

**Discussion by tomtom** [gsantner/markor#1415] [Q&A] [answered]
## General information

* App version: 2.8.0

### Description

Most editory keep a list of most recently used files (editing history). Am I right that markor misses this feature?

Could you please add a list of most recently used files somewhere? When I open a file from that list the file's folder should become the current working folder in the files list - if possible.

This is the feature I miss the most from markor. Other than that markor is a really cool markdown editor.

**Comment by tomtom** [comment]
Ok I found it in that icon that looks like a folder. So my request rather is about discoverability of this feature.

**Reply by tomtom** [reply]
Nevermind, it's ok the way it is.

**Comment by gsantner** [comment, marked as answer]
Click the folder button in the file browser's top menu. There you can find recent files, popular files and more.

PRO tipp: you can also long-press the `+` button to quickly move to the recent folder.

**Reply by tomtom** [reply]
I have to admit I never examined what this button is for. Thanks for the hints.
